api/graphql/tests/test_reservations/test_reservation_create.py

```diff
@@ -1107,6 +1109,7 @@ def test_create_price_calculation_with_fixed_price_reservation_unit(
             price_unit=PriceUnit.PRICE_UNIT_FIXED,
             lowest_price=1.0,
             highest_price=3.0,
+            highest_price_net=Decimal(3) / (1 + tax_percentage.value / 100),
```

Comment:
We are doing this dividing by hundred in many places. I wonder if it would make sense to add another field to `tax_percentage` that returns the decimal value directly? That way we could simplify this to:
```python
highest_price_net=Decimal(3) / 1 + tax_percentage.decimal_value)
```

Reply:
this would be nice convenience method / property 👍 I'll add it!